[2023-05-05 01:09:32,235] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: postgres_to_localfile.write_suppliers_to_file manual__2023-05-05T01:09:27.197787+00:00 [queued]>
[2023-05-05 01:09:32,246] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: postgres_to_localfile.write_suppliers_to_file manual__2023-05-05T01:09:27.197787+00:00 [queued]>
[2023-05-05 01:09:32,247] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2023-05-05 01:09:32,247] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2023-05-05 01:09:32,248] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2023-05-05 01:09:32,264] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): write_suppliers_to_file> on 2023-05-05 01:09:27.197787+00:00
[2023-05-05 01:09:32,267] {standard_task_runner.py:52} INFO - Started process 4071 to run task
[2023-05-05 01:09:32,271] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'postgres_to_localfile', 'write_suppliers_to_file', 'manual__2023-05-05T01:09:27.197787+00:00', '--job-id', '63', '--raw', '--subdir', 'DAGS_FOLDER/load_from_postgres.py', '--cfg-path', '/tmp/tmp609l20s3', '--error-file', '/tmp/tmpj6w68m9k']
[2023-05-05 01:09:32,273] {standard_task_runner.py:77} INFO - Job 63: Subtask write_suppliers_to_file
[2023-05-05 01:09:32,359] {logging_mixin.py:109} INFO - Running <TaskInstance: postgres_to_localfile.write_suppliers_to_file manual__2023-05-05T01:09:27.197787+00:00 [running]> on host ebd7ac88a2d5
[2023-05-05 01:09:32,406] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.7/site-packages/***/models/xcom.py:323 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2023-05-05 01:09:32,425] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.7/site-packages/***/models/renderedtifields.py:126 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2023-05-05 01:09:32,433] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.7/site-packages/***/models/renderedtifields.py:162 SAWarning: Coercing Subquery object into a select() for use in IN(); please pass a select() construct explicitly
[2023-05-05 01:09:32,445] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.7/site-packages/***/models/renderedtifields.py:163 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2023-05-05 01:09:32,450] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=***@example.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=postgres_to_localfile
AIRFLOW_CTX_TASK_ID=write_suppliers_to_file
AIRFLOW_CTX_EXECUTION_DATE=2023-05-05T01:09:27.197787+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-05-05T01:09:27.197787+00:00
[2023-05-05 01:09:32,466] {base.py:79} INFO - Using connection to: id: postgres_local. Host: host.docker.internal, Port: 5432, Schema: sample_source_db, Login: ***, Password: ***, extra: {}
[2023-05-05 01:09:32,474] {sql.py:375} INFO - Running statement: SELECT * FROM suppliers, parameters: None
[2023-05-05 01:09:32,476] {sql.py:384} INFO - Rows affected: 5
[2023-05-05 01:09:32,477] {python.py:175} INFO - Done. Returned value was: None
[2023-05-05 01:09:32,496] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=postgres_to_localfile, task_id=write_suppliers_to_file, execution_date=20230505T010927, start_date=20230505T010932, end_date=20230505T010932
[2023-05-05 01:09:32,522] {local_task_job.py:154} INFO - Task exited with return code 0
[2023-05-05 01:09:32,572] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.7/site-packages/***/models/dagrun.py:926 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2023-05-05 01:09:32,576] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-05-05 01:09:32,578] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.7/site-packages/***/models/dag.py:1076 SADeprecationWarning: Query.value() is deprecated and will be removed in a future release.  Please use Query.with_entities() in combination with Query.scalar() (deprecated since: 1.4)
[2023-05-05 01:09:32,579] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.7/site-packages/***/models/dag.py:1076 SADeprecationWarning: Query.values() is deprecated and will be removed in a future release.  Please use Query.with_entities() (deprecated since: 1.4)
